.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: 2
+
+jobs:
+  build:
+    working_directory: ~/ember-get-component
+    parallelism: 1
+
+    docker:
+    - image: circleci/node:6.14-stretch-browsers
+
+    environment:
+      APP_NAME: ember-get-component
+
+    steps:
+    - checkout
+    - run:
+        name: Update PATH variable
+        command: echo 'export PATH=""${PATH}:${HOME}/.yarn/bin:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin""' >> ${BASH_ENV}
```

Comment:
Do you know that this step is necessary?

1) I'd be surprised if `yarn` isn't already in the PATH in this image
2) It shouldn't be necessary to append `node_modules/bin` to the PATH, that's what `yarn run` is for. For example `yarn run bower install`